src/main/resources/static/js/createSurvey.js

```diff
@@ -64,13 +86,33 @@ $("#createSurveyForm").on('click', '#addTextQuestion', function () {
      addNewQuestion(prompt, inputType)
 });
 
+// Button listener to add range question to evaluation to array for submission later
+$("#createSurveyForm").on('click', '#addRangeQuestion', function () {
+     prompt = $('#rangeQuestionPrompt').val()
+     inputType =  $(".inputSelect").val()
+	 min = $('#rangeQuestionMin').val()
+	 max = $('#rangeQuestionMax').val()
+	 if(isNaN(min) || isNaN(max)  ){
```

Comment:
I think these if-else blocks are backwards. Also, I think a more strong check than `isNaN` might be appropriate here. `isNaN("3.14") = false`

Reply:
Thanks to Sam's suggestion, I will no longer need to check if its a number.  Curious, why did you think the if/elses were backwards?  I thought checking whether the inputs were numbers would take priority over whether they were equivalent/not in order

Reply:
```
if (not a number){
    keep going
} else {
    alert
}
```

Reply:
Ah gotcha, I misunderstood what you meant.  Thankfully its no longer relevant